en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md

```diff
@@ -20,29 +21,60 @@ This section explains as to how APIs in WSO2 API Manager can be secured using mu
 
 
 5.  Provide an alias and public certificate. Select the tier that should be used to throttle out the calls using this particular client certificate and click **Upload**.
-    [![]({{base_path}}/assets/img/learn/upload-certificate.png)]({{base_path}}/assets/img/learn/upload-certificate.png)
     
-6.  **Save** the API
+     [![]({{base_path}}/assets/img/learn/upload-certificate.png)]({{base_path}}/assets/img/learn/upload-certificate.png)
+    
+6.  **Save** the API.
     
 ### Invoke an API secured with Mutual SSL using Postman
```

Comment:
Is the overall TOC correct?

<img width="219" alt="Screenshot 2020-08-31 at 12 33 24" src="https://user-images.githubusercontent.com/5195851/91691961-3543c400-eb86-11ea-8fcf-d2a08e853413.png">